### PR TITLE
Fix 9534

### LINKED ISSUE #9534: For moderators, Show the presence of a CIQUAL/Agribalyse match in the panel for ingredient analysis
### Problem
- Make it easy to spot holes in Agribalyse/CIQUAL coverage, and easier to look for proxies, without overwhelming all users

### Proposed solution
- For moderators, Show the presence of a CIQUAL/Agribalyse match in the panel for ingredient analysis

### Part of
- #6499

### PR DESCRIPTION
**Pull request by himanshisrestha**
### What
This will issue #9534 , in which we should show which ingredients have a ciqual_food_code or a ciqual_proxy_food_code.
The file Display.pm in lib/ProductOpener had the function named "display_ingredients_analysis_details" which was calling for a function named "display_nested_list_of_ingredients". In the function, it searches for the properties for the ingredient in which the required fields (ciqual_food_code or a ciqual_proxy_food_code) if present was added to the string formed to get the desired output.

### Screenshot

Screenshot 2024-03-19 at 8 50 18 PM

### Related issue(s) and discussion

- Fixes #9534

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -10992,7 +10992,10 @@ sub display_nested_list_of_ingredients ($ingredients_ref, $ingredients_text_ref,
 		${$ingredients_list_ref}
 			.= "<li>" . "<span$class>" . $ingredient_ref->{text} . "</span>" . " -> " . $ingredient_ref->{id};
 
-		foreach my $property (qw(origin labels vegan vegetarian from_palm_oil percent_min percent percent_max)) {
+		foreach my $property (
+			qw(origin labels vegan vegetarian from_palm_oil percent_min percent percent_max ciqual_proxy_food_code ciqual_food_code)
```
Comment (stephanegigandet) [inline lib/ProductOpener/Display.pm:10996]: Good, can you change the order a bit:

```suggestion
			qw(origin labels vegan vegetarian from_palm_oil ciqual_food_code ciqual_proxy_food_code percent_min percent percent_max)
```

**Review by stephanegigandet — COMMENTED** [review]
@himanshisrestha Thank you! Could you remove the other changes from this PR? (the one that were for import template spreadsheet), and the .env file

**Comment by himanshisrestha** [comment]
> @himanshisrestha Thank you! Could you remove the other changes from this PR? (the one that were for import template spreadsheet), and the .env file

Yes I can

File: lib/ProductOpener/Display.pm (hunk 1 above)
Comment (alexgarel) [reply lib/ProductOpener/Display.pm:10996]: Initial demand was to display it only for moderators. I don't know if it's important (@teolemon ?)

**Pull request closed without merging**